### ISSUE TITLE
Add any basic Java Program and star this repo

### ISSUE THREAD

**Issue by VedantKCSE** [VedantKCSE/Java-Programs#1]
Create a folder of your name 
then add you code

### **"How to contribute in repo and what is  PR ?"**
This is the main **AGENDA** Of this repository.

**Comment by deepesh85b** [comment]
@VedantKCSE 
Please assign this to me

**Comment by VedantKCSE** [comment]
> @VedantKCSE Please assign this to me

You can PR Directly

**Comment by Premjeet1804** [comment]
can i do this one sir

**Comment by VedantKCSE** [comment]
you can just pr @Premjeet1804

**Comment by Premjeet1804** [comment]
remote: Permission to VedantKCSE/Java-Programs.git denied to Premjeet1804.
fatal: unable to access 'https://github.com/VedantKCSE/Java-Programs.git/': The requested URL returned error: 403

it is showing this sir

**Comment by Premjeet1804** [comment]
can u please guide me

**Comment by Premjeet1804** [comment]
i really want to make my first PR

**Comment by VedantKCSE** [comment]
> can u please guide me

You Need to fork it first

**Comment by VedantKCSE** [comment]
> yeah i did it

Then just add your edits and add PR

**Comment by Premjeet1804** [comment]
let me try again'
sir

**Comment by Premjeet1804** [comment]
* my-new-feature
PS C:\Users\hp\Desktop\issue\Java-Programs> git push
To push the current branch and set the remote as upstream, use

    git push --set-upstream origin my-new-feature

PS C:\Users\hp\Desktop\issue\Java-Programs> git merge my-new-feature
Already up to date.
PS C:\Users\hp\Desktop\issue\Java-Programs> git push origin
To push the current branch and set the remote as upstream, use

    git push --set-upstream origin my-new-feature


still facing these

**Comment by Premjeet1804** [comment]
i'm sorry  i'm new to these

**Comment by VedantKCSE** [comment]
Hey You can just use GUI instead of commandline 

> i'm sorry i'm new to these

**Comment by Premjeet1804** [comment]
yeah i was succesfull

**Comment by Premjeet1804** [comment]
thankyou so much sir..
now you can check my PR..

**Comment by Subhajit767** [comment]
please assign this to me

**Comment by Aishaa-26** [comment]
@VedantKCSE Please assign this to me... I am new to open source contribution, hence want to try this as a starter.

**Comment by VedantKCSE** [comment]
> @VedantKCSE Please assign this to me... I am new to open source contribution, hence want to try this as a starter.
> 

You can PR directly 😄

**Comment by Subhajit767** [comment]
I want to work on this so please assign this to me

**Comment by VedantKCSE** [comment]
> I want to work on this so please assign this to me

You can add PR Directly

**Comment by VedantKCSE** [comment]
> Can you assign this issue to me?

PR Directly
